Replace randint() with user-chosen method.

tcld opened the pull request:
This is a very small change to the Python implementation of the BaseNoise-class. I chose the parameter-method over the subclass-method, both suggested by @caseman . I chose this one because I think the parameter-solution basically covers the other one, too (create subclass of BaseNoise, add custom randint-method, use it to call BaseNoise-constructor). And it can work even without subclassing BaseNoise.

If there are any further changes I should make or if I missed anything, please let me know.

caseman commented:
Can we call the argument/instance var randint_function instead of randint_substitute? Other than that it looks good.

tcld commented:
I renamed it. Hoping to make the use a bit more convenient I also added this: https://github.com/caseman/noise/pull/10/files#diff-9a0810b4e11652c8b44090f9e7388e8eR102
